src/api/constants.js

```diff
@@ -0,0 +1,11 @@
+/* @flow strict-local */
+
+/**
+ * The topic servers understand to mean "there is no topic". This should
+ * match
+ *   https://github.com/zulip/zulip/blob/5.0-dev/zerver/lib/actions.py#L2714
+ * or similar logic at the latest `main`.
+ */
```

Comment:
nit: one-line summary:

```suggestion
/**
 * The topic servers understand to mean "there is no topic".
 *
 * This should match
 *   https://github.com/zulip/zulip/blob/5.0-dev/zerver/lib/actions.py#L2714
 * or similar logic at the latest `main`.
 */
```